ietf/templates/meeting/activity_report.html

```diff
@@ -1,32 +1,22 @@
-{% extends "base.html" %}
-{% load ams_filters ietf_filters cache %}
-{% block title %}IETF {{ meeting.number }} Proceedings - Progress Report{% endblock %}
-{% block content %}
-    {% cache 3600 proceedings_progress_report meeting.number %}
-    <h1>
-        <a class="text-decoration-none text-reset"
-           href="{% url 'ietf.meeting.views.proceedings' num=meeting.number %}">
-            IETF {{ meeting.number }} proceedings
-        </a>
-    </h1>
-    <h2 class="mt-3">IETF Progress Report</h2>
-    <h3 class="mt-3">{{ sdate|date:"d-F-y" }} to {{ edate|date:"d-F-y" }}</h3>
+    {% load ams_filters ietf_filters %}
     <ul class="progress-section">
         <li>{{ actions_count }} IESG Protocol and Document Actions this period</li>
         <li>{{ last_calls_count }} IESG Last Calls issued to the IETF this period</li>
         <li>
             {{ new_drafts_count|stringformat:"3s" }}    New I-Ds   ({{ new_drafts_updated_count }} of which were updated, some ({{ new_drafts_updated_more_count }}) more than once)
         </li>
         <li>{{ updated_drafts_count|stringformat:"3s" }}    I-Ds were updated   (Some more than once)</li>
-        <li>
-            <h3 class="mt-3">In the final 4 weeks before meeting</h3>
-        </li>
-        <li>
-            {{ ffw_new_count|stringformat:"3s" }}    New I-Ds were received - {{ ffw_new_percent }} of total newbies since last meeting
-        </li>
-        <li>
-            {{ ffw_update_count|stringformat:"3s" }}    I-Ds were updated - {{ ffw_update_percent }} of total updated since last meeting
-        </li>
+        {% if not is_monthly_report %}
```

Comment:
This raises the question of whether "is_monthly_report" accurately describes the condition being passed in. Reading this, I suspect what we're really trying to tell the template is "This is a meeting report". How is the parameter otherwise used?

Reply:
Right now the distinction is binary, this template fragment is either used in a monthly activity report or a meeting activity report (spanning all time between two meetings). So "not is_monthly_report" is equivalent to "is_meeting_report". I added is_monthly_report because that was the new view for this PR. I can instead add "is_monthly_report" which is potentially more specific. Alternatively we could simply check for the existence of the meeting template variable, which isn't passed from the monthly activity report view. But if it were added later, the side effect would be unwanted.

Reply:
Switched to use is_meeting_report in commit 9d762819d13b996da3059a05ea5ccfbe026e3947